resources/install.rb

```diff
@@ -0,0 +1,33 @@
+property :upgrade_strategy, equal_to: %w(checkout sync), default: 'checkout'
```

Comment:
If you want to make it so you can just say

```ruby
ruby_build_install'
```

You can do this trick (which Foodcritic alerts on)

```ruby
property :name, String, default: ''
```

Reply:
I don't think I want to do this, this time 🤔